client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-connect-account.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Image from 'components/image';
+import resizeImageUrl from 'lib/resize-image-url';
+
+class StripeConnectAccount extends Component {
+	static propTypes = {
+		stripeConnectAccount: PropTypes.shape( {
+			displayName: PropTypes.string,
+			email: PropTypes.string,
+			firstName: PropTypes.string,
+			isActivated: PropTypes.bool,
+			lastName: PropTypes.string,
+			logo: PropTypes.string,
+		} ),
+		onDisconnect: PropTypes.func, // TODO - require most of these props in subsequent PR
+	}
+
+	renderLogo = () => {
+		const { stripeConnectAccount } = this.props;
+		const { logo } = stripeConnectAccount;
+
+		let image = null;
+
+		if ( ! isEmpty( logo ) ) {
+			image = <Image src={ resizeImageUrl( logo, { w: 40, h: 40 } ) } className="stripe__connect-account-logo" />;
+		}
+
+		return (
+			<div className="stripe__connect-account-logo-container">
+				{ image }
+			</div>
+		);
+	}
+
+	renderNameAndEmail = () => {
+		const { stripeConnectAccount } = this.props;
+		const { displayName, email, firstName, lastName } = stripeConnectAccount;
+		const name = ! isEmpty( displayName ) ? displayName : `${ firstName } ${ lastName }`;
+
+		return (
+			<div className="stripe__connect-account-details">
+				<span className="stripe__connect-account-name">
+					<span className="stripe__connect-account-name">{ name }</span>
+					<span className="stripe__connect-account-email">{ email }</span>
+				</span>
+			</div>
+		);
+	}
+
+	// TODO - when we are ready to connect this for-reals, this layer may not be needed
+	onDisconnect = ( event ) => {
+		event.preventDefault();
+		if ( this.props.onDisconnect ) {
+			this.props.onDisconnect();
+		}
+	}
+
+	renderStatus = () => {
+		const { stripeConnectAccount, translate } = this.props;
+		const { isActivated } = stripeConnectAccount;
+
+		let status = null;
+
+		if ( isActivated ) {
+			status = <span className="stripe__connect-account-status account-activated">
+				{ translate( 'Activated' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **56** times:
`translate( 'Activated?' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/17002-stripe-connect-ux)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).